pkg/controller/handlers/knowledgefile/knowledgefile.go

```diff
@@ -30,12 +32,19 @@ func (u *UnsupportedError) Error() string {
 type Handler struct {
 	invoker   *invoke.Invoker
 	gptScript *gptscript.GPTScript
+	limit     int
 }
 
 func New(invoker *invoke.Invoker, gptScript *gptscript.GPTScript) *Handler {
+	defaultLimit := env.VarOrDefault("OTTO_KS_INGESTION_LIMIT", "1000")
```

Comment:
Can this be an actual configuration parameter? In other words, can this be added here: https://github.com/otto8-ai/otto8/blob/0582592a8fc318a1ce96d60218e43df0e34a040f/pkg/services/config.go#L50-L61

Then passed as a parameter to this function.

Reply:
yeah sure